tap-schema/sql/0004_tap_schema.permissions.sql

```diff
@@ -0,0 +1,7 @@
+
+-- grant usage on schema tap_schema to ''@'%';
+
+GRANT SELECT ON tap_schema.* TO 'TAP_SCHEMA'@'%';
+FLUSH PRIVILEGES;
+
+ALTER USER 'TAP_SCHEMA'@'%' IDENTIFIED WITH mysql_native_password BY 'TAP_SCHEMA';
```

Comment:
This is where this would have to be updated somehow to get the password from an environment variable to address the comment on https://github.com/lsst-sqre/charts/pull/415. My recollection is that there's a mechanism in the MySQL Docker image to add a hook script instead of a SQL file that would have access to environment variables and thus could read an injected password. I think you just install a `.sh` file instead of a `.sql` file.